series/supportedseries.go

```diff
@@ -223,6 +221,24 @@ var ubuntuSeries = map[string]SeriesVersionInfo{
 		Supported:    false,
 		ESMSupported: false,
 	},
+	"kinetic": {
+		Version:   "22.10",
+		Supported: false,
+	},
+	"lunar": {
+		Version:   "23.04",
+		Supported: false,
+	},
+	"mantic": {
+		Version:   "23.10",
+		Supported: false,
+	},
+	"noble": {
+		Version:      "24.40",
+		LTS:          true,
+		ESMSupported: true,
```

Comment:
We have to be careful here because we can end up allowing a controller or workload being able to use noble.


https://github.com/juju/juju/blob/a9666e918ccca7ef5c06b0528a94137869a32d22/core/series/supported.go#L130-L143

https://github.com/juju/juju/blob/a9666e918ccca7ef5c06b0528a94137869a32d22/core/series/supported.go#L149-L161

Reply:
Also at the time (this is 4+ years ago) I didn't have the courage to bring on the os/series package into Juju, which would have made this so much easier to work out. Today we should 100% just bring in the os package into Juju (it's on my very long list).

Reply:
Ah yea, good point, I've changed that to false, that matches the others in there.

Reply:
Yea it defo should be brought in, I can imagine we're not even using most of it, like all this reading distroinfo stuff will not be needed at all.